src/pymovements/transforms.py

```diff
@@ -287,30 +285,23 @@ def cut_into_subsequences(
     idx = 0
     for t in range(0, arr.shape[0]):
         for i in range(0, n):
-            # cut out 1000 ms piece of trial t
+            # get piece of respective sequence
```

Comment:
if we have a comment here I think this should be more specific. e. g. get piece of sequence with shape (window_size, ...)

Reply:
or part of the sequence / subsequence